packages/schemas/alterations/next-1731054001-init-account-center.ts

```diff
@@ -4,6 +4,11 @@ import type { AlterationScript } from '../lib/types/alteration.js';
 
 const alteration: AlterationScript = {
   up: async (pool) => {
+    // Delete the default account center if it exists
+    await pool.query(sql`
+      delete from account_centers where id = 'default';
+    `);
```

Comment:
nit: you can log the deleted row count for a better debugging experience.

Reply:
updated